docs/logging-configuration.asciidoc

```diff
@@ -0,0 +1,105 @@
+ifdef::env-github[]
+NOTE: For the best reading experience,
+please view this documentation at https://www.elastic.co/guide/en/apm/agent/ruby[elastic.co]
+endif::[]
+
+[[logging-configuration]]
+== Logging configuration
+
+The Ruby agent allows you to set logging options as well as configure trace/log correlation.
```

Comment:
Perhaps add something like _"This makes it possible to jump directly from your APM data to your log data in Kibana, if both are in the same ES cluster (or however it works with a link to possible documentation?)"_

Reply:
sure, I will add that.

Reply:
Do we have documentation for this somewhere else besides in agents?